README.md

```diff
@@ -1 +1,41 @@
-# portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+"Portfolio" project
+> Portfolio Project is going to be an open project which began now to the end of training. In this Project
+
```

Comment:
- [ ] Optional
    * Let people know what your project can do specifically. Provide context and add a link to any reference visitors might be unfamiliar with.